datasets/haalcentraal/bag/dataset.json

```diff
@@ -0,0 +1,395 @@
+          "oppervlakte",
+          "status"
+        ],
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "identificatie": {
+            "type": "string",
+            "description": "Dit is de identificatie van een verblijfsobject, standplaats of ligplaats."
+          },
+          "domein": {
+            "type": "string",
+            "description": "Het domein waartoe de identificatie behoort."
+          },
+          "type": {
+            "type": "string",
+            "enum": [
+              "verblijfsobject",
+              "standplaats",
+              "ligplaats"
+            ]
+          },
+          "documentdatum": {
+            "type": "string",
+            "format": "date",
+            "description": "De vaststellingsdatum van het brondocument dat de basis is voor opname, wijziging of een verwijdering van een object."
+          },
+          "documentnummer": {
+            "type": "string",
+            "description": "De unieke aanduiding van het brondocument op basis waarvan een opname, mutatie of een verwijdering van gegevens ten aanzien van een woonplaats heeft plaatsgevonden, binnen een gemeente. Alle karakters uit de MES-1 karakterset zijn toegestaan."
+          },
+          "gebruiksdoelen": {
+            "type": "string",
+            "enum": [
+              "woonfunctie",
+              "bijeenkomstfunctie",
+              "celfunctie",
+              "gezondheidszorgfunctie",
+              "industriefunctie",
+              "kantoorfunctie",
+              "logiesfunctie",
+              "onderwijsfunctie",
+              "sportfunctie",
+              "winkelfunctie",
+              "overige gebruiksfunctie"
+            ]
+          },
+          "geconstateerd": {
+            "type": "boolean",
+            "description": "Indicatie dat een standplaats, ligplaats of verblijfsobject in de registratie is opgenomen door een feitelijke constatering, zonder dat er een brondocument aan ten grondslag ligt. Het adresseerbaar object is mogelijk illegaal."
+          },
+          "geometrie": {
+            "$ref": "https://geojson.org/schema/Geometry.json",
+            "description": "Een samengesteld geometriegegevenstype waarbij wordt afgedwongen dat voor de geometrie een keuze gemaakt moet worden tussen een punt of een vlak."
+          },
+          "pandIdentificaties": {
+            "type": "array",
+            "description": "Identificatie(s) van het pand of de panden waar het verblijfsobject deel van is.",
+            "items": {
+              "type": "string"
+            }
+          },
+          "nummeraanduidingIdentificaties": {
+            "type": "array",
+            "description": "Identificatie(s) van de hoofd- en nevenadressen van de standplaats, ligplaats of verblijfsobject.",
+            "items": {
+              "type": "object",
```

Comment:
Could this be a relation, or doesn't that work for a proxied dataset?

Reply:
That doesn't work.